index.html

```diff
@@ -413,22 +423,27 @@
         </td>
         <!-- Requirements and Bait -->
         <td>
+          <!-- Put Fishers intuition if fish has at least 1 predetor -->
           {{?it.bait.hasPredators}}
             <div class="ui middle aligned status-icon sprite-icon sprite-icon-status-intuition" title="Fisher's Intuition"></div>
           {{?}}
+          <!-- Put fishEyes if fishEyes is not false and if true enter fishEyesDuration()  -->
           {{?it.fishEyes !== false}}
             <div class="ui middle aligned status-icon sprite-icon sprite-icon-status-fish_eyes {{?it.fishEyes !== true}}has-duration{{?}}" title="Fish Eyes">
               {{?it.fishEyes !== true}}
                 <span>{{=it.fishEyesDuration()}}</span>
               {{?}}
             </div>
           {{?}}
+          <!-- Put snagging if fish has snagging != null -->
```

Comment:
These are kinda redundant don't you think?  The if statement makes it pretty obvious.

Reply:
Yeah these can be removed (i put it in for some codefolding on my pc to have a better overview)